src/drive/web/modules/upload/UploadButton.jsx

```diff
@@ -15,6 +15,7 @@ const UploadButton = ({ label, disabled, onUpload, className }) => (
     multiple
     onChange={onUpload}
     data-test-id="upload-btn"
+    value={[]} // always erase the value to be able to re-upload the same file
```

Comment:
should we put this fix in UI?

Reply:
That's where I put it at first but I'm not sure. If we use `hidden={false}`, we don't want it but I think we may want it in all other cases. It depends of the desired behavior in fact. If the UI displays the value in some way, it seems OK that nothing happens if the same file is put in the input (that's the behavior of the browser's widget)